react/css/src/digitv2/components/panelCardV2.scss

```
@@ -54,91 +54,91 @@
               /* Media query for tablets */
-              font-size: theme("digitv2.fontSize.body-s.tablet");
+              font-size: var( --digitv2-fontSize-body-s-tablet);
             }
         
             @media (min-aspect-ratio: 3/4) {
               /* Media query for desktop */
-              font-size: theme("digitv2.fontSize.body-s.desktop");
+              font-size: var( --digitv2-fontSize-body-s-desktop);
             }
-            color: theme("digitv2.lightTheme.generic-inputborder");
+            color: var(--digitv2-lightTheme-generic-inputborder);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)